search/src/Components/upload/react-images-upload/index.js

```diff
@@ -88,6 +91,15 @@ class ReactImageUploadComponent extends React.Component {
         this.setState({fileErrors});
         continue;
       }
+      // Check for duplication
+      if (!this.props.images.includes(file.name) || filesName.includes(file.name)){
+        fileError = Object.assign(fileError, {
+          type: ERROR.FILE_DUPLICATE
```

Comment:
In the previous logic, this case did not trigger an error, it was just skipped.

Reply:
Yeah it's skipped but the user can see an error message saying certain image is duplicate

Reply:
In the existing logic, or only in your changed version?

Reply:
In the current version of this branch

Reply:
We definitely don't want this as an actual error if it will stop us clicking Next...?
But I think it's best not to report it as an error in any case. Better just to skip a file that's already been uploaded.

Reply:
Seems to work pretty well - received duplicate errors, but the process just continued.